### ISSUE TITLE
Feature Suggestion: adding option to the classification to remove rows where target variable is empty/null

### ISSUE THREAD

**Issue by sparvaneh** [pycaret/pycaret#249]
Hi,

Current version consider an empty cell in the target as a new class. It would be great to add an option to allow removing rows where target class is empty/null.

**Comment by pycaret** [comment]
Good suggestion @sparvaneh.

Tagging this as an enhancement for future release.

**Comment by pycaret** [comment]
@Yard1 I think we can easily do this in the `setup` function before `train_test_split` in the new structure. We can drop all the rows where the target variable is `NULL` (applicable to `pycaret.classification` and `pycaret.regression`). We will also have to throw some warning in the logger for that since we are modifying the number of rows of the original dataset.

**Comment by Yard1** [comment]
Alternatively, we could add it as a preprocessing pipeline step. I think it would be better that way, as it will stay consistent, and also work on passed data.

**Comment by pycaret** [comment]
@Yard1 Yes. The reason I suggested in the `setup` is that it's not really `preprocessing`. Its basically preparing data for preprocessing as we do not need a `transformer` for this. If we can put it on top in `preprocess.py`, that's fine as well.

**Comment by pycaret** [comment]
@Yard1 Yes. This is already implemented. Great, let me test it.
